olm-catalog/serverless-operator/project.yaml

```diff
@@ -38,9 +38,9 @@ dependencies:
 
   eventing_kafka: 1.1.0
```

Comment:
Should we remove this for 1.25?

Reply:
QE is still using this for testing, let's keep it, it's only used for building the source image.